ipld/get_shares.go

```diff
@@ -82,11 +92,13 @@ func GetShares(ctx context.Context, bGetter blockservice.BlockGetter, root cid.C
 					// leaf has its own additional leaf(hack) so get it
 					nd, err := plugin.GetNode(ctx, bGetter, lnks[0].Cid)
 					if err != nil {
-						// again, we don't care
+						// again, we don't really care much, just fetch as much as possible
+						span.RecordError(err)
```

Comment:
Let's set status on error and on success, as per https://opentelemetry.io/docs/reference/specification/trace/api/#set-status

An example: https://uptrace.dev/opentelemetry/go-tracing.html#setting-status-code

Reply:
Alright, tried this, is it what you had in mind?

Reply:
Yes